targets/TARGET_STM/TARGET_STM32F4/TARGET_STM32F411xE/device/TOOLCHAIN_IAR/stm32f411xe.icf

```diff
@@ -18,9 +18,9 @@ define region ROM_region = mem:[from __region_ROM_start__ to __region_ROM_end__]
 define region RAM_region = mem:[from __region_RAM_start__ to __region_RAM_end__];
 
 /* Stack and Heap */
-/*Heap 1/4 of ram and stack 1/8*/
-define symbol __size_cstack__ = 0x4000;
-define symbol __size_heap__   = 0x8000;
+/*Heap 84kB and stack 8kB */
+define symbol __size_cstack__ = 0x2000;
```

Comment:
There's alignment for stack to be 1k @mprse  please confirm

Reply:
I confirm, stack size should be 1k..